bbot/presets/spider-intense.yml

```diff
@@ -0,0 +1,17 @@
+description: Recursive web spider with more aggressive settings
+
+modules:
```

Comment:
including the existing spider preset here will let us maintain the module list / blacklist in one place

Reply:
https://github.com/blacklanternsecurity/bbot/pull/2205/commits/695442e65d264e23d0a7e81d1eb355359543f955